include/stflrichtext.h

```diff
@@ -0,0 +1,42 @@
+#ifndef NEWSBOAT_STFLSTRING_H_
+#define NEWSBOAT_STFLSTRING_H_
+
+#include <map>
+#include <string>
+
+namespace newsboat {
+
+class StflRichText {
```

Comment:
Indeed. I think the commit message is slightly incorrect. The `Stfl::quote()` method seems to be only necessary in very specific places, but that has no effect on this PR's changes (which is about the Richtext "quoting").

I will try to find a better name to differentiate these (to avoid using "quoting" for different concepts), however, I prefer to apply such a renaming in a different PR.